libs/filabridge/include/filament/MaterialChunkType.h

```diff
@@ -45,6 +45,7 @@ enum UTILS_PUBLIC ChunkType : uint64_t {
     MaterialMetal = charTo64bitNum("MAT_METL"),
     MaterialShaderModels = charTo64bitNum("MAT_SMDL"),
     MaterialSamplerBindings = charTo64bitNum("MAT_SAMP"),   // no longer used
+    MaterialProperties = charTo64bitNum("MAT_PROP"),
```

Comment:
We should probably deprecate the `MaterialSpecularAntiAliasing` chunk and instead use this new one to determine if the material has specular anti-aliasing.

Reply:
No this is just for properties set inside the shader. specularAA is different.

Reply:
Ah I see